<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgoritmoSencillo</class>
 <widget class="QWidget" name="AlgoritmoSencillo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algoritmo Sencillo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 221, 207);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Britannic Bold</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ALGORITMO 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Britannic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ORDENAR PALABRAS ALFABÃ‰TICAMENTE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="tiempo1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tiempo2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tiempo3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tiempo4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tiempo5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_t1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Britannic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tiempo 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_t2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Britannic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tiempo 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_t3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Britannic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tiempo 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_t4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Britannic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tiempo 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_t5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Britannic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tiempo 5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushEjecutar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string>EJECUTAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushReset">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>RESETEAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_media">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Britannic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 127);</string>
   </property>
   <property name="text">
    <string>MEDIA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="media">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(126, 255, 173);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushFichero">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona fichero...</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sinOrdenar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>104</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="ordenadas">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>104</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Palabras sin ordenar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Palabras ordenadas</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
